From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Sun, 3 Mar 2019 10:22:34 -0500
Subject: gpgv: Improve documentation for keyring choices

* doc/gpgv.texi: Improve documentation for keyring choices

--

From the existing documentation, it's not clear whether the default
keyring will always be mixed into the set of keyrings, or whether it
will be skipped if a --keyring is present.  The updated text here
attempts to describe the keyring selection logic more completely.

Signed-off-by: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
(cherry picked from commit a7b2a87f940dba078867c44f1f50d46211d51719)
---
 doc/gpgv.texi | 15 ++++++++-------
 1 file changed, 8 insertions(+), 7 deletions(-)

diff --git a/doc/gpgv.texi b/doc/gpgv.texi
index a052861..2dd9576 100644
--- a/doc/gpgv.texi
+++ b/doc/gpgv.texi
@@ -59,13 +59,14 @@ no configuration files and only a few options are implemented.
 That does also mean that it does not check for expired or revoked
 keys.
 
-By default a keyring named @file{trustedkeys.kbx} is used; if that
-does not exist a keyring named @file{trustedkeys.gpg} is used.  The
-default keyring is assumed to be in the home directory of GnuPG,
-either the default home directory or the one set by an option or an
-environment variable.  The option @code{--keyring} may be used to
-specify a different keyring or even multiple keyrings.
-
+If no @code{--keyring} option is given, @code{gpgv} looks for a
+``default'' keyring named @file{trustedkeys.kbx} (preferred) or
+@file{trustedkeys.gpg} in the home directory of GnuPG, either the
+default home directory or the one set by the @code{--homedir} option
+or the @code{GNUPGHOME} environment variable.  If any @code{--keyring}
+option is used, @code{gpgv} will not look for the default keyring. The
+@code{--keyring} option may be used multiple times and all specified
+keyrings will be used together.
 
 @noindent
 @mansect options
